llvm/include/llvm/Analysis/ScalarEvolution.h

```diff
@@ -640,6 +640,9 @@ class ScalarEvolution {
   /// \p IndexExprs The expressions for the indices.
   LLVM_ABI const SCEV *
   getGEPExpr(GEPOperator *GEP, const SmallVectorImpl<const SCEV *> &IndexExprs);
```

Comment:
```suggestion
  getGEPExpr(GEPOperator *GEP, ArrayRef<const SCEV *> IndexExprs);
```

for uniformity with the other one, while we're touching these lines -- shouldn't have any use-site-visible difference, right?